AjaxForm.php

```diff
@@ -105,9 +104,8 @@ protected function jumpToOrReload($intId, $strParams = null, $strForceLang = nul
      */
     private static function sendResponse($content)
     {
-        $insertTags = new \Contao\InsertTags();
-        $content = $insertTags->replace($content, false);
-        $objResponse = new HtmlResponse($content);
-        $objResponse->send();
+        $content = \Contao\System::getContainer()->get('contao.insert_tag.parser')->replaceInline($content);
```

Comment:
the `contao.insert_tag.parser` service does not exist in Contao 4.9 (which is supported according to `composer.json`)!

Reply:
This pull request has been already merged, and this problem already fixed afterward: https://github.com/terminal42/contao-ajaxform/commit/17cd66cb2acef3b7ff3e8adf0dcc9b367bdc4abc